Check_Balance.py

```diff
@@ -0,0 +1,16 @@
+import CalcSeparators
+
+def Check_Balance():
+    userID = int(input("Please enter your user ID: "))
+    with open(r"accountfiles.txt","r") as file:
+        f = file.readlines()
+        if userID > len(f) or userID < 1:
+            print("Error: That User ID does not exist")
+            return
+        details = f[userID-1]
+        #This function calculates the positions of the separators in the line of text
+        Separators = CalcSeparators.CalcSeparators(userID,details)
```

Comment:
My poor eyes... (If all the functions are in utils.py we don't need to CalcSeparators.CalcSeparators() just use CalcSeparators() and if you use it in the main file you just use utils.CalcSeparators())

Reply:
Haha I will be sure to make a utils files, I like this idea a lot more than having tons of files